src/Paramore.Brighter/CommandProcessor.cs

```diff
@@ -461,6 +472,166 @@ public async Task PublishAsync<T>(
             }
         }
 
+        /// <inheritdoc />
+        public string SchedulerPost<TRequest>(TRequest request, 
```

Comment:
As above. I suggest we overload Post with a method that takes a DateTimeOffset ```at``` which is the time at which you want to publish. We could have a ```TimeSpan delay``` but I assume with most schedulers that ends up as now + timespan. Perhaps a helper extension for TimeSpan of TimeSpan.FromX.Delay() where delay just adds to now? Either way I think of a scheduler as being an At, but you might want an easy way to mean that as fromNow